jbmc/src/jbmc/jbmc_parse_options.cpp

```diff
@@ -800,7 +802,7 @@ int jbmc_parse_optionst::get_goto_program(
       show_goto_functions(
         *goto_model,
         log.get_message_handler(),
-        ui_message_handler.get_ui(),
+        message_handler.get_ui(),
```

Comment:
🍌 Again, are `log.get_message_handler()` and `message_handler` different? It looks quite confusing.